confidential-data-hub/docs/use-cases/secure-mount-with-block-device.md

```diff
@@ -0,0 +1,77 @@
+# Secure mount with Block Device
+
+This guide helps an user to use confidential data hub to secure mount inside TEE environment.
+
+## Preliminaries
+
+- Ensure that [cryptsetup](https://gitlab.com/cryptsetup/cryptsetup/) is installed.
+
+## Example
+
+### Create a loop device
+
+```shell
+$ loop_file="/tmp/test.img"
+$ sudo dd if=/dev/zero of=$loop_file bs=1M count=1000
+$ sudo losetup -fP $loop_file
+$ device=$(sudo losetup -j $loop_file | awk -F'[: ]' '{print $1}')
+$ echo $device
+# Output should be something like /dev/loop0
+$ device_num=$(sudo lsblk -no MAJ:MIN $device)
+$ echo $device_num
+# Output should be something like 7:0
+```
+
+### Secure mount inside a TEE environment
+
+1. Build the CDH and its client tool
+
+Follow the instructions in the [CDH README](../../README.md#confidential-data-hub) and [Client Tool README](../../README.md#client-tool) to build the CDH and its client tool.
+
+2. Install `luks-encrypt-storage`
+
+Install [luks-encrypt-storage](../../storage/scripts/luks-encrypt-storage) into `/usr/local/bin`
```

Comment:
As a separate service, CDH could require install extra things. I am thinking about whether we can integrate the installations of `luks-encrypt-storage`, `cachefs` and `ossfs` binaries/scripts in CDH's Makefile. In CCv0, we explicit install them [in kata scripts](https://github.com/kata-containers/kata-containers/pull/7965/files#diff-4a3807c6c399f5750341b6dd154009c87f1dc69fc0345b66a0af381ebdf1dab4R79-R80). This would bring extra maintaining cost for kata. cc @fitzthum @fidencio

Reply:
I agree that we can move the tasks of installing binaries or scripts to CDH, as they are only used in the CDH. CDH is responsible for installing them, and we can simplify the code in kata.